src/statisticBlock.js

```diff
@@ -128,9 +148,9 @@ class StatisticBlock extends Component {
            && size != this.props.options.size)
         ))
       {
-        if(element.clientHeight < scrollHeight
-          || element.clientHeight < childHeight
-          || element.clientWidth < scrollWidth) {
+        if( // removed element.clientHeight < scrollHeight ||
+          element.clientHeight < childHeight
+          || element.clientWidth < childWidth) {
```

Comment:
Will this catch the case when the content grows larger than its container (i.e. causing overflow) or is it no longer an issue?